### PR TITLE
Added GP Maven plugin user guide

### PR DESCRIPTION
**Pull request by yumaoka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,16 @@ information. For example, exporting a resource bundle hosted by an instance
 of Globalization Pipeline service to a specific file format, adding a new target
 language in bundle configuration, creating a new user and so on.
 
-To see available commands and options, please see [Globalization Pipeline Command Line Interface Guide](gp-cli.md)
+To see available commands and options, please see [Globalization Pipeline CLI Tool User Guide](gp-cli.md)
+
+## Maven Plugin (gp-maven-plugin)
+
+This plugin integrates Globalization Pipeline service with Apache Maven build.
```
Comment (srl295) [inline README.md]: `with Apache` --> `with an Apache…`

File: gp-maven-plugin.md [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+
+For example, you have translatable English *.properties files under `src/nls1/resources/en`
+and `src/nls2/resources/en`. If you want to include both sets, use the configuration below.
+
+```
+    <configuration>
+        <bundleSets>
+            <bundleSet>
+                <sourceFiles>
+                    <direcotry>src/nls1/resources/en</directory>
```
Comment (srl295) [inline gp-maven-plugin.md]: typo - `directory`

File: gp-maven-plugin.md [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+                    <includes>
+                        <include>**/*.properties</include>
+                    </includes>
+                </sourceFiles>
+            </bundleSet>
+            <bundleSet>
+                <sourceFiles>
+                    <direcotry>src/nls2/resources/en</directory>
```
Comment (srl295) [inline gp-maven-plugin.md]: typo - `directory`

File: gp-maven-plugin.md [hunk 4]
```diff
@@ -0,0 +1,544 @@
...
+            [...]
+        </plugins>
+    </build>
+    [...]
+</project>
+```
+
+### <a name="TOC-Usage-Credentials"></a>Specifying Globalization Pipeline Service Credentials
+
+The plugin requires *service instance administrator* credentials for a Globalization
```
Comment (srl295) [inline gp-maven-plugin.md]: could link to https://github.com/IBM-Bluemix/gp-common#quick-start-guide here

Comment (srl295) [inline gp-maven-plugin.md]: could link to https://github.com/IBM-Bluemix/gp-common#quick-start-guide here

File: README.md (hunk 1 above)
Comment (srl295) [inline README.md]: `with Apache` --> `with an Apache`…

File: gp-maven-plugin.md (hunk 3 above)
Comment (srl295) [inline gp-maven-plugin.md]: typo: `direcotry`

File: gp-maven-plugin.md (hunk 2 above)
Comment (srl295) [inline gp-maven-plugin.md]: typo: `direcotry`

File: gp-maven-plugin.md [hunk 5]
```diff
@@ -0,0 +1,544 @@
...
+    </targetLanguages>
+```
+
+If this parameter is not specified, `upload` goal creates a new Globalization Pipeline bundle
+with all available machine translation target languages, and `download` goal exports all
+target languages currently available in the Globalization Pipeline bundle.
+
+#### `<languageMap>`
+
+Specifies custom language mappings. Each nested element name is a BCP 47 language tag used by
```
Comment (srl295) [inline gp-maven-plugin.md]: link: https://tools.ietf.org/html/bcp47 (minor)

File: gp-maven-plugin.md (hunk 2 above)
Comment (yumaoka) [inline gp-maven-plugin.md]: corrected.

File: gp-maven-plugin.md (hunk 3 above)
Comment (yumaoka) [inline gp-maven-plugin.md]: corrected.

File: gp-maven-plugin.md (hunk 4 above)
Comment (yumaoka) [inline gp-maven-plugin.md]: Added one more sentence with the link.

File: README.md (hunk 1 above)
Comment (yumaoka) [inline README.md]: changed.

File: gp-maven-plugin.md (hunk 5 above)
Comment (yumaoka) [inline gp-maven-plugin.md]: Added the link

**Pull request merged**